docs/css/footer.css

```diff
@@ -0,0 +1,63 @@
+#footer {
```

Comment:
I would say, remove the id footer here, because it's empty you don't need it, right?

Reply:
Yeah, you are right.